### PR TITLE
add jupyterhub package

### PR DESCRIPTION
**Pull request by vigsterkr**

### CODE REVIEW

**Comment by joerg84** [comment]
@vigsterkr LGTM could you add a short example to https://github.com/dcos/examples (and add a link to that in your description).

File: repo/packages/J/jupyterhub/0/config.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+              }
+            },
+            "required": [
+              "var_name",
+              "value"
+            ]
+          }
+        },
+        "config_file": {
+          "description": "Locaiton of the jupyter_config.py to be used for the service.",
```
Comment (joerg84) [inline repo/packages/J/jupyterhub/0/config.json]: Location

File: repo/packages/J/jupyterhub/0/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "minDcosReleaseVersion": "1.7",
+  "name": "jupyterhub",
+  "version": "0.7.2-0.1",
+  "tags": ["mesosphere", "framework"],
+  "maintainer": "viktor.gal@maeth.com",
+  "description": "Multi-user server for Jupyter notebooks",
+  "scm": "https://github.com/jupyterhub/jupyterhub",
+  "website": "http://jupyter.org/",
+  "framework": true,
+  "preInstallNotes": "This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Experimental packages should never be used in production!",
```
Comment (joerg84) [inline repo/packages/J/jupyterhub/0/package.json]: Maybe add that it requires RexRay/external volume support configured.

**Comment by vigsterkr** [comment]
@joerg84 thnx for getting back to me. actually i did couple of changes during yesterday so i'll push them soon, but the problem is still with `env_vars` config variable :S
although it's a valid jsonschema, dcos UI does not let me to set anything like this as value:
```
[{"var_name": "foo", "value": "bar"}]
```
any suggestions?

there's really just too much environmental variable to be exposed...

**Comment by vigsterkr** [comment]
regarding the example i'll write a blog article about this because this is a bigger project with dcos + jupyterhub :)

**Comment by vigsterkr** [comment]
but sure i'll prepare some basic examples as well :)

**Comment by ryadav88** [comment]
@vigsterkr any update here?

**Comment by vigsterkr** [comment]
@ryadav88 currently there's an issue with marathon when specifying external volumes for the apps created by jupyterhub, see https://jira.mesosphere.com/browse/MARATHON-7102

**Comment by ryadav88** [comment]
thanks for the update. Adding a WIP label to this PR for now.

**Comment by joerg84** [comment]
@vigsterkr Any news here?

**Pull request closed without merging**